integrations/access/accessrequest/app.go

```diff
@@ -234,12 +234,20 @@ func (a *App) onPendingRequest(ctx context.Context, req types.AccessRequest) err
 		return trace.Wrap(err)
 	}
 
+	loginsByRole, err := a.getLoginsByRole(ctx, req)
+	if trace.IsAccessDenied(err) {
+		log.Warnf("failed to get logins by role. error: %s", err)
```

Comment:
I think we can be more explicit about what needs to be done to have this error fixed.
We already know this was an access denied, so saying something like this can help the user fix the issue.
```suggestion
		log.Warnf("Missing permissions to get logins by role. Please add role.read to the associated role. error: %s", err)
```